app/templates/index.hbs

```diff
@@ -44,7 +46,11 @@
                 <div class="col-xs-12">
                     <h2>{{t "index.subjects.heading"}}</h2>
```

Comment:
Is this supposed to still have a subject heading? 
<img width="967" alt="screen shot 2017-06-13 at 2 19 59 pm" src="https://user-images.githubusercontent.com/6268982/27097778-6c2f255e-5043-11e7-85bf-7b3c77e04f76.png">